lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,42 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    const startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), Number.MAX_VALUE);
```

Comment:
phew! I can achieve it using a hypothetical combo of stylesheets returning a 500 (which Chrome will re-request!?) and HTML imports injected later with a delayed script just as the first blocking stylesheet comes in :) but glad that situation isn't common in practice haha